### PR TITLE
Adding TF Lite variants of the EAST model

### PR DESCRIPTION
**Pull request by sayakpaul**
EAST model (as proposed in https://arxiv.org/abs/1704.03155) for text detection from natural scenes.

### CODE REVIEW

File: tfhub_dev/assets/sayakpaul/east-text-detector/int8/1.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Lite sayakpaul/east-text-detector/int8/1
+TF Lite quantized version of the EAST model for text detection from natural scenes.
+
+<!-- parent-model: sayakpaul/east-text-detector/1 -->
+<!-- asset-path: https://github.com/sayakpaul/Adventures-in-TensorFlow-Lite/releases/download/v0.6.0/east_model_float16.tar.gz -->
```
Comment (maringeo) [inline tfhub_dev/assets/sayakpaul/east-text-detector/int8/1.md:5]: Should this be int8?

Comment (sayakpaul) [reply tfhub_dev/assets/sayakpaul/east-text-detector/int8/1.md]: Yes. Fixed it up.

**Comment by maringeo** [comment]
Thank you @sayakpaul for contributing to TF Hub!

The models are live on tfhub.dev: https://tfhub.dev/sayakpaul/lite-model/east-text-detector/dr/1

This PR has been merged in commit 2c9c7deb7c86324bbfc5c76c61ef9527ebb9e2ca.

**Comment by sayakpaul** [comment]
Thanks, @maringeo.

**Comment by sayakpaul** [comment]
@maringeo I made a typo in the one liner description here ("natural"). It should have been "natural". Could you fix it up or would you suggest me to create a PR for this?

**Comment by maringeo** [comment]
Hi @sayakpaul, thanks for spotting that. I've made the change and it should be live on Github and on tfhub.dev in the next 24 hours.

**Comment by sayakpaul** [comment]
Thank you, @maringeo!

**Pull request closed without merging**